src/components/SocialBtns/fb_sdk.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable */
+module.exports = {
+  init: function() {
+    window.fbAsyncInit = function() {
+      FB.init({
+        appId      : '339880856413925',
```

Comment:
This follows [facebook quick start instructions](https://developers.facebook.com/docs/javascript/quickstart). The appID is a unique identifier for the app. We also have an app secret that can't be shared. With that said should maybe we should create a separate config file with all keys and ids of external resources . We could also pull out the email used for trello in webpack/parts.json and put it in config.